### PR TITLE
Database tutorial

### PR DESCRIPTION
**Pull request by micahjohnson150**
Here is my placeholder for the database tutorial. I will be adding more jupyter notebooks. Happy to make changes.

### CODE REVIEW

**Comment by micahjohnson150** [comment]
Some of these issues are on my end. I figured the notebooks were static. I was messing with the DB right after I pushed so some of the db rows hadn't made it back in yet.

**Comment by scottyhq** [comment]
Thanks for opening this up @micahjohnson150 ! It's looking good! For the notebooks to render on the website, we have to be able to read from the database via github actions. So you'll have to use the database *external IP rather than the internal IP which would only be accessible from jupyterhub*. If you need to add authentication (username and password) that can be done by adding GitHub Secrets environment variables, but If I remember correctly the plan is to just allow anonymous access for the hackweek, in which case that won't be necessary.

**Comment by scottyhq** [comment]
@micahjohnson150 the 'ensure clean notebooks' test failed. https://github.com/snowex-hackweek/website/pull/37/checks?check_run_id=2731836685 . this is to make sure we don't commit notebooks with output to the repository (b/c image outputs are big and changes get ugly). If you could please do a `Kernel -> Restart kernel and clear all outputs` then re-commit your notebooks we can get this merged

**Comment by micahjohnson150** [comment]
Ah! I see. I was wondering about that error. 

So @scottyhq I should always save the notebooks after clearing the kernel and then commit those?  e.g. there should be no output in my notebook?

**Comment by scottyhq** [comment]
> I should always save the notebooks after clearing the kernel and then commit those? e.g. there should be no output in my notebook? 

Exactly! JupyterBook executes the notebooks and stores rendered HTML on a separate branch (gh-pages) which is what is displayed on the website.

File: book/tutorials/database/1_getting_started_example.ipynb [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    "- Cross referencing data\n",
+    "- Enables GIS functionality\n",
+    "- Ready for use in your code\n",
+    "- Provenance!\n",
+    "- Ready for use in a GIS software like ArcGIS or QGIS!\n",
+    "\n",
+    "### TL;DR Do less wrangling, do more crunching. \n",
+    "\n",
+    "\n",
+    "# What is it exactly?\n",
```
Comment (scottyhq) [inline book/tutorials/database/1_getting_started_example.ipynb:29]: In order for the sidebar navigation to work as expected you want only one header per file and the rest subheaders:
```
```
# Introduction to the SnowEx Database
## Why a database?
### TL;DR Do less wrangling, do more crunching.
## What is it exactly?
```
```
